io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,55 @@
+id: io.neovim.nvim.tool.tmux
+build-extension: true
+sdk: org.freedesktop.Sdk//22.08
+runtime: io.neovim.nvim
+runtime-version: stable
+branch: stable
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/tools/tmux
+  prepend-pkg-config-path: /app/tools/tmux/lib/pkgconfig
+  ldflags: -L/app/tools/tmux/lib
+  cflags: -I/app/tools/tmux/include
+modules:
+  - name: libevent
+    buildsystem: autotools 
+    config-opts:
+      - --enable-shared
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+  - name: libutempter
+    buildsystem: simple
+    build-commands:
+      - make -C libutempter libdir=${FLATPAK_DEST}/lib libexecdir=${FLATPAK_DEST}/lib includedir=${FLATPAK_DEST}/include mandir=${FLATPAK_DEST}/share/man
```

Comment:
you use `subdir: libutempter`

Reply:
Fixed https://github.com/flathub/flathub/pull/4119/commits/2bf009bff34e871d40b18700d0414bd91a795c23